### PR TITLE
Spelling Mistakes Correction

### PR DESCRIPTION
**Pull request by ENUMERA8OR**
Corrected common spelling mistakes across the following files:

docs/meshery line 185
docs/README.md line 66
docs/Gemfile.lock lines 160, 204, 217 & 404
docs/v0.4/guides.html line 874
docs/v0.4/functionality.html line 859
docs/v0.4/es.html line 752
docs/v0.4/service-meshes.html line 738
docs/v0.4/reference.html line 869
docs/v0.4/extensibility.html line 650
docs/v0.4/404.html line 656
docs/v0.4/overview.html line 766
docs/v0.4/concepts.html line 613

### CODE REVIEW

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -401,7 +401,7 @@ GEM
       io-console (~> 0.5)
     rexml (3.2.8)
       strscan (>= 3.0.9)
-    rouge (3.26.0)
+    rogue (3.26.0)
```
Comment (leecalcote) [inline docs/Gemfile.lock:404]: Thanks, however, this package came must remain the same.

File: docs/README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Alternatively, if you are running Windows 10, you may install the Windows Subsys
 - Theme - https://github.com/vsoch/docsy-jekyll
 </details>
 
-# Meshery Documentation Stucture
+# Meshery Documentation Structure
```
Comment (leecalcote) [inline docs/README.md:66]: Good 👍

File: docs/meshery [hunk 3]
```diff
@@ -182,7 +182,7 @@ else
   fi
 fi
 
-#Transfering permissions and ownership to client USER
+#Transferring permissions and ownership to client USER
```
Comment (leecalcote) [inline docs/meshery:185]: Nice catch

File: docs/Gemfile.lock (hunk 1 above)
Comment (ENUMERA8OR) [reply docs/Gemfile.lock:404]: Oh! Sorry my bad...

**Comment by l5io** [comment]
🚀 Preview for commit 3a02c897b9f501662fadb5c24ec15b40c187be8c at: https://668de883fdccce38806c2627--meshery-docs-preview.netlify.app

File: docs/Gemfile.lock (hunk 1 above)
Comment (ENUMERA8OR) [reply docs/Gemfile.lock]: Updated the corrections

**Comment by l5io** [comment]
🚀 Preview for commit 84a37111307d18c7e222a262e8858c729b5a54ee at: https://668de9f76c695738e810a03c--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@ENUMERA8OR 
Let's discuss this in Meshery Dev meeting starting 7:30 PM IST Today. Add this as an agenda item to the meeting minutes, if you would :)
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

**Comment by l5io** [comment]
🚀 Preview for commit a196c87098469514fe849f566c0fda085f5ac119 at: https://668e21bc0dd7fa71e1ada656--meshery-docs-preview.netlify.app

**Comment by ENUMERA8OR** [comment]
Sorry but I have to be somewhere at that time its important. I will not be able to attend the meet today...

**Comment by l5io** [comment]
🚀 Preview for commit 42a79bb41bbc7b1b78499902c77c5122865df584 at: https://668e25a68aa78b734fddf8a3--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a7ec14d05a258ae9fb504f5cea5771e59c068ad6 at: https://668e27b3dfc64477a2b8d818--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**